docs/zopebook/ZopeServices.rst

```diff
@@ -340,25 +340,32 @@ Now you can use the new MailHost object from a Script.
 Error Logging Services
 ----------------------
 
-The *Site Error Log* object, typically accessible in the Zope root
-under the name 'error_log', provides debugging and error logging
-information in real-time.  When your site encounters an error, it
-will be logged in the Site Error Log, allowing you to review (and
-hopefully fix!) the error.
+.. note::
 
-Options settable on a Site Error Log instance
-include:
+  As of Zope 4, the **Site Error Log** is no longer a a dependency of Zope,
+  but has to be installed additionally if you want to use it.
```

Comment:
Replace „additionally“ with „separately“

Reply:
Do we have a section which describes how to install Zope Products?
At least the actual name of the package (`Products.SiteErrorLog`) should be mentioned.

Reply:
@dataflake Thanks!

@icemac Added the package name and added a link to PyPi.

Yes, there is section which explains how to install a product, but.. oh boy... this needs some **polish**.

https://zope.readthedocs.io/en/latest/zopebook/MaintainingZope.html#installing-new-products